mne_bids/path.py

```diff
@@ -56,40 +56,53 @@ class BIDSPath(object):
     space : str | None
         The coordinate space for an anatomical file. Corresponds to "space".
     split : int | None
-        The split of the continuous recording file for ``.fif`` data. Corresponds to "split".
+        The split of the continuous recording file for ``.fif`` data.
+        Corresponds to "split".
     prefix : str | None
         The prefix for the filename to be created. E.g., a path to the folder
         in which you wish to create a file with this name.
     kind : str | None
-        The filename kind. This is the entity after the last ``_`` before the extension. 
-        E.g., ``'ieeg'``.
+        The filename kind. This is the entity after the
+        last ``_`` before the extension. E.g., ``'ieeg'``.
     extension : str | None
         The extension of the filename. E.g., ``'.json'``.
+    check : bool
+        If True enforces the entities to be valid according to the
+        current BIDS standard. Defaults to True.
+
+    Attributes
+    ----------
+    check : bool
+        If True enforces the entities to be valid according to the
+        current BIDS standard.
```

Comment:
Should say here that this check is performed upon init or update method calls only.

Reply:
But when else **could** they be performed? 🧐

Reply:
a naive user may think that upon setting an attribute, some weird (and non-existing) magic performs the check.

```Python
# perform a check!
myobj.check=True
# why is nothing happening?
```

Reply:
But in that case they'll just have to learn that this is not how Python code usually works :)